packages/synapse-react-client/src/components/SchemaDrivenAnnotationEditor/AnnotationEditorUtils.ts

```diff
@@ -27,6 +27,20 @@ export function dropNullishArrayValues(
   return newFormData
 }
 
+export function dropNullValues(
```

Comment:
Synapse does not allow annotations to have a `null` value, so add a utility to fix that